connectors/es/sink.py

```diff
@@ -203,6 +209,9 @@ async def run(self):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Sink...")
             raise
+        except asyncio.QueueFull as e:
+            msg = "Connector was unable to ingest data into overloaded Elasticsearch. Make sure Elasticsearch instance is healthy, has enough resources and content index is healthy."
+            raise ElasticsearchOverloadedError(msg) from e
```

Comment:
I am not a python expert, but I _think_ that if you `raise` inside an `except`, you automagically get the `__cause__` set for you without having to make a special constructor or use the `from e`

Reply:
They are chained automatically, but I personally like `from e` as being more explicit